traffic_portal/app/src/common/api/DeliveryServiceUriSigningKeysService.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+var DeliveryServiceUriSigningKeysService = function(Restangular, messageModel) {
+	this.getKeys = function(id) {
+		var keys = Restangular.withConfig(function(cfg) {
+			cfg.setBaseUrl('/api/1.3');
```

Comment:
fyi, there is some discussion around what version this should be. we may just move to v2.2 to reflect the version of TC. stay tuned.

Reply:
Aye. I'm not a fan of this particular work-around, but it works for now. When we pick a new version, we can update all the things at once.